web/book/src/reference/stdlib/transforms/filter.md

```diff
@@ -18,7 +18,16 @@ from employees
 filter (age > 25 || department != "IT")
 ```
 
+```prql
+from employees
+filter (department | in ["IT", "HR"])
+```
+
 ```prql
 from employees
 filter (age | in 25..40)
 ```
+
+```admonish note
+You can also use `in` operator to [declare variables or functions](../compute)
```

Comment:
I think maybe this is a bit superfluous, or possibly we could phrase it differently. Because any function can be used as part of a variable or function, no? There's nothing specific to `in`?

Reply:
I tend to agree. I tried to follow @aljazerzen comment but maybe I didn't understand properly what was expected